src/blkdrv/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+if(WITH_BLUESTORE)
+set(libblkdrv_srcs
+    BlockDevice.cc)
+endif(WITH_BLUESTORE)
+
+if(HAVE_LIBAIO OR HAVE_POSIXAIO)
+  list(APPEND libblkdrv_srcs
+    kernel_drv/KernelDevice.cc
+    kernel_drv/ceph_io_uring.cc
```

Comment:
why do we need to add the "ceph" prefix?

Reply:
You can check
https://github.com/ceph/ceph/pull/34622#discussion_r412193869
Which options do your prefer? A or B or C?
A. keep using the original file name aio.cc & io_uring.cc
B. aio.cc -> ceph_aio.cc
C. aio.cc->ceph_aio.cc & io_uring.cc -> ceph_io_uring.cc

Reply:
I think that once we move headers to some fixed dir, it will fix header name collision, won't it?
I imagine ```#include "block/aio.h``` will be ok.

Reply:
> 
> 
> I think that once we move headers to some fixed dir, it will fix header name collision, won't it?
> I imagine `#include "block/aio.h` will be ok.

@aclamk 
The reason why it is called `ceph_aio.h`, because I ran into conflits on FreeBSD, which actually has `aio.h` for its POSIX AIO implementation. 
So I renamed it to `ceph_aio.h`, and from that stems my suggestion to also rename `aio.cc` to make naming consistent, and the name of the header file matches that of the src name.

And TBH it rather have building error out, than it including the wrong header files.
Moving it to `block/aio.h` will make the include more specific, and I think that the renaming can be undone.